plugins/in_tail/win32/stat.c

```diff
@@ -35,6 +35,8 @@
  */
 
 #define UINT64(high, low) ((uint64_t) (high) << 32 | (low))
+#define LDAP_TO_SECONDS_DIVISOR 10000000
```

Comment:
Please rename this constant so it's clear at first sight that it's the conversion ratio between windows ticks and seconds.